cloudinit/sources/DataSourceLXD.py

```diff
@@ -331,9 +348,16 @@ def read_metadata(
                 md[LXD_SOCKET_API_VERSION]["config"][cfg_key] = response.text
                 # Promote common CONFIG_KEY_ALIASES to top-level keys.
                 if cfg_key in CONFIG_KEY_ALIASES:
```

Comment:
Nit: Do we want an `else` on this `if` to raise if the parsed key doesn't exist in our ALIASES? Probably not necessary, but would help indicate a problem in our handling of the datasource.